### ISSUE TITLE
When to update the Nodejs version to v16.6.2, v14.17.5 or v12.22.5 

### ISSUE THREAD

**Issue by jimmyjasai56** [alpinelinux/docker-alpine#198]
I'm sorry, I want to know how often the alpine will update the Nodejs version, there have some security updates I need to use them.

**Comment by Yonle** [comment]
By the way, The latest node release can installed by installing `node-current`

**Comment by jimmyjasai56** [comment]
@Yonle Thanks for the mention. So I can install the v12.22.5 version by node-current?

**Comment by Yonle** [comment]
I don't think that nodejs version v12 is still exist on alpine package repositories. All what i know is:

`nodejs` gives node v14
`nodejs-currect` gives currect stable nodejs release (`v16.6.0`)

You may also use https://nvm.io if you want to try another node version (including older version)

**Comment by jimmyjasai56** [comment]
@Yonle Nodejs 12 https://dl-cdn.alpinelinux.org/alpine/v3.12/main/x86_64/

**Comment by Yonle** [comment]
Yeah you're right. It's node v12 for x86_64
Maybe using [`nvm`](https://nvm.sh) would solve this problem

**Comment by jimmyjasai56** [comment]
@Yonle Yeah nvm is good.  But in the alpine image, I must install a lot of dependencies for nvm. I hope the alpine team can upgrade the version.

**Comment by Yonle** [comment]
I hope so.
Currently i got node v16 on alpine arm64
